### ISSUE TITLE
questions about dataset

### ISSUE THREAD

**Issue by kejia1** [Algolzw/image-restoration-sde#79]
I want to know how to create the GT and LQ datasets in train? Do GT and LQ use norain*.png and rain*.png?

**Comment by Algolzw** [comment]
Yes! You just need to move/copy all clean images into GT and put all low-quality images into the LQ directory.

**Comment by kejia1** [comment]
thanks,I am running your code
